conbench/entities/run.py

```diff
@@ -249,6 +247,30 @@ def get_default_baseline_run(
 
         return Run.get(closest_run_id)
 
+    def get_candidate_baseline_runs(self) -> Dict[str, Optional[_FoundBaselineRun]]:
+        """Return information about a few different candidate baseline runs, including
+        on the parent commit, fork-point commit, and head-of-default-branch commit.
+
+        See docstring of _get_baseline_run_from_baseline_commit() for how these are
+        found.
+        """
+        return {
+            "parent": self._get_baseline_run_from_baseline_commit(
+                self.commit.get_parent_commit()
+            ),
+            "fork_point": self._get_baseline_run_from_baseline_commit(
+                self.commit.get_fork_point_commit()
+            ),
+            "head_of_default": self._get_baseline_run_from_baseline_commit(
+                Session.execute(
+                    s.select(Commit)
+                    .filter(Commit.sha == Commit.fork_point_sha)
+                    .order_by(s.desc(Commit.timestamp))
+                    .limit(1)
```

Comment:
`HEAD` is a moving target, and depends on perspective.

This here is the "newest commit on the default branch that Conbench has gotten a result for"?

Reply:
Yes! :) perhaps "head_of_default" is the wrong phrase.

Reply:
I renamed to `latest_default`. Does that make more sense or is it more confusing?

Reply:
That sounds good to me. It's slightly ambiguous if it's "the latest commit" or "the latest commit with benchmarks" but I doubt we can get out of that without a lot more words